units/oldest_container_removal.go

```diff
@@ -0,0 +1,110 @@
+package units
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/cloud"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/pkg/errors"
+)
+
+const (
+	oldestImageJobName = "oldest-image"
```

Comment:
I feel like this needs a verb, maybe "oldest-image-removal." Also the file name is inconsistent with this